pysat/_constellation.py

```diff
@@ -238,10 +236,111 @@ def add(self, bounds1, label1, bounds2, label2, bin3, label3,
         xarr = np.arange(numx)
         binx = None
 
-        # TODO modify output
-        out_2d = _calc_2d_median(ans, data_label, binx, biny, xarr, yarr,
-                                 zarr, numx, numy, numz)
-
+        ############################################
```

Comment:
If we copy this routine in here, it can be defined within this class function as a routine.  This would make updates simpler.

Reply:
removing these routines as discussed in meeting